bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -63,9 +63,28 @@ func GetKfApp(kfdef *kfdefsv2.KfDef) kftypes.KfApp {
 	return _coordinator
 }
 
-func getConfigFromCache(pathDir string, kfDef *kfdefsv2.KfDef) ([]byte, error) {
+func getStaticConfigForPlatform(kfDef *kfdefsv2.KfDef) string {
```

Comment:
What's the purpose of this function? Shouldn't any config  go into the static file?

Reply:
The idea is:
* If the user has specified a config file, that file is used.
* If the user hasn't specified a config file, the config value can be seeded with a static config file for the specific platform.

Reply:
I'm not sure I follow. Isn't the idea to get rid of the paltform flag and replace it with a static config file?

i.e. instead of specifying --platform=gcp, I'd specify 
```
--config=kfctl_gcp_istio.yaml
```

To put it another way. What if we made it an error to specify both "--platform" and "--config"? Would we still need this function?

Reply:
i think command completion should provide a list of static config files available to help the user
eg
--config kfctl_<tab><tab> kfctl_default.yaml kfctl_default_istio.yaml kfctl_gcp.yaml kfctl_gcp_istio.yaml, ...